extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/FormAuthRuntimeConfig.java

```diff
@@ -124,4 +125,51 @@ enum CookieSameSite {
      * The default value is empty, which means the cookie will be kept until the browser is closed.
      */
     Optional<Duration> cookieMaxAge();
+
+    /**
+     * The post location.
+     */
+    @WithDefault("/j_security_check")
+    String postLocation();
+
+    /**
+     * One-time authorization code configuration.
+     */
+    @WithName("otac")
```

Comment:
```suggestion
    @WithName("one-time-authentication-token")
```

Assuming we agree to replace `authorization-code` with `authentication-token`

Reply:
I used `authentication-token` instead, the one-time thing is bit too verbose